package.json

```diff
@@ -37,7 +37,7 @@
     "handlebars": "^4.7.6",
     "is": "^3.1.0",
     "is-ip": "^3.1.0",
-    "isolated-vm": "^3.3.3",
+    "isolated-vm": "^2.0.1",
```

Comment:
should we fix the isolate-vm version?
also if we are not committing package-lock file, should we remove it from the source?

Reply:
We should commit package-lock.json file unless we have a good reason not to. 
package-lock should automatically take care of this.